### ISSUE TITLE
HSTS Problem

### ISSUE THREAD

**Issue by gnebbia** [argos83/ritm#24]
I tried the example, and generated my own certificate, but with firefox I still have the problem 
"This site uses HTTP Strict Transport Security (HSTS) to specify that Firefox may only connect to it securely. As a result, it is not possible to add an exception for this certificate."

Any idea on how to fix this?

**Comment by argos83** [comment]
Yes, you've got to install the CA you generated in firefox.. did you do
that? If you do it correctly, you'll see the green lock and there will be
no need to add an exception

**Comment by gnebbia** [comment]
Wait I added on firefox the certificate in the preferences and importing it in the "Authorities" section, and i can view it from there, but still I have this warnings, what am i doing wrong?
When i generate the certificate maybe I am inserting wrong infos? I'm just inserting random strings when it asks for info about name, city and so on.

What the problem could be due to?

Thanks
